Sources/OAuth2Swift.swift

```diff
@@ -290,3 +289,13 @@ open class OAuth2Swift: OAuthSwift {
     }
 
 }
+
+extension OAuth2Swift {
+
+    public enum CallbackURLEncode {
```

Comment:
Ooops, forgot about it. I've tried using enum as suggested, but realised that naming might be an issue and would cause issues for users, who already use the initial property, so went back to boolean forgetting to remove the declaration. Well noticed.